ui/src/app/base/selectors/pod/pod.test.js

```diff
@@ -72,10 +72,6 @@ describe("pod selectors", () => {
       name: "podrick",
       id: 222,
     });
-    expect(pod.getById(state, "222")).toStrictEqual({
```

Comment:
Removed this test because it was only useful when using `useParams` which returned the pod id as a string. The id type should always be number when running this function.